picard/util/tags.py

```diff
@@ -166,652 +227,1128 @@ def names(self, selector=None):
         'djmixer',
         shortdesc=N_('DJ-Mixer'),
+        longdesc=N_('The names of the DJ mixers for the track.'),
     ),
     TagVar(
         'encodedby',
         shortdesc=N_('Encoded By'),
+        longdesc=N_('The person or organization that encoded the track.'),
+        not_from_mb=True,
     ),
     TagVar(
         'encodersettings',
         shortdesc=N_('Encoder Settings'),
+        longdesc=N_('The settings used when encoding the track.'),
+        not_from_mb=True,
     ),
     TagVar(
         'engineer',
         shortdesc=N_('Engineer'),
+        longdesc=N_('The names of the engineers associated with the track.'),
     ),
     TagVar(
         'extension',
         shortdesc=N_('File extension'),
+        longdesc=N_("The file's extension."),
         is_hidden=True,
         is_preserved=True,
         is_tag=False,
+        not_from_mb=True,
     ),
     TagVar(
         'file_created_timestamp',
         shortdesc=N_('File created timestamp'),
+        longdesc=N_('The file creation timestamp in the form "YYYY-MM-DD HH:MM:SS" as reported by the file system.'),
         is_hidden=True,
         is_preserved=True,
         is_tag=False,
+        not_from_mb=True,
     ),
     TagVar(
         'file_modified_timestamp',
         shortdesc=N_('File modified timestamp'),
+        longdesc=N_('The file modification timestamp in the form "YYYY-MM-DD HH:MM:SS" as reported by the file system.'),
         is_hidden=True,
         is_preserved=True,
         is_tag=False,
+        not_from_mb=True,
     ),
     TagVar(
         'filename',
         shortdesc=N_('File name'),
+        longdesc=N_('The name of the file without extension.'),
         is_hidden=True,
         is_preserved=True,
         is_tag=False,
+        not_from_mb=True,
     ),
     TagVar(
         'filepath',
         shortdesc=N_('File Path'),
+        longdesc=N_('Full path and name of the file.'),
         is_hidden=True,
         is_tag=False,
+        not_from_mb=True,
     ),
     TagVar(
         'filesize',
         shortdesc=N_('File size'),
+        longdesc=N_('Size of the file in bytes.'),
         is_file_info=True,
         is_hidden=True,
         is_tag=False,
+        not_from_mb=True,
     ),
     TagVar(
         'format',
         shortdesc=N_('File format'),
+        longdesc=N_('Media format of the file (e.g.: MPEG-1 Audio).'),
         is_file_info=True,
         is_hidden=True,
         is_preserved=True,
         is_tag=False,
+        not_from_mb=True,
     ),
     TagVar(
         'gapless',
         shortdesc=N_('Gapless Playback'),
+        longdesc=N_('Indicates whether or not there are gaps between the recordings on the release.'),
+        not_from_mb=True,
     ),
     TagVar(
         'genre',
         shortdesc=N_('Genre'),
+        longdesc=N_(
+            'A multi-value tag containing the specified genre information from MusicBrainz. Requires the '
+            '"Use genres from MusicBrainz" option setting be enabled.'
```

Comment:
I think we should list required options by name and use title property to link them to the tag. See https://github.com/metabrainz/picard/blob/36014f0a6e93ab80edf3ab221e0bea5a5209770f/picard/options.py#L243

If a tag depends on one (or more) options, it could be convenient to be able to make the link programmatically, and it limits need to re-translate strings if an option title changes.


```python

TagVar(

   related_options = ('use_genres', ),
```